tests/unit/tests/Framework/QueryBuilder/AggregateTest.php

```diff
@@ -0,0 +1,250 @@
+<?php
+
+namespace unit\tests\Framework\QueryBuilder;
+
+use Give\Framework\Database\DB;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @unreleased
+ *
+ * @covers Aggregate
+ */
+final class AggregateTest extends TestCase
+{
+    /**
+     * Truncate posts and give_donationmeta table to avoid duplicate records
+     *
+     * @unreleased
+     *
+     * @return void
+     */
+    public function tearDown()
+    {
+        parent::tearDown();
+
+        $posts        = DB::prefix('posts');
+        $donationMeta = DB::prefix('give_donationmeta');
+
+        DB::query("TRUNCATE TABLE $posts");
+        DB::query("TRUNCATE TABLE $donationMeta");
+    }
+
+    /**
+     * @unreleased
+     *
+     * @return void
+     */
+    public function testCount()
```

Comment:
@alaca for test methods, let's make sure to be as obnoxiously descriptive as possible.  So instead of `testCount` - we'll want to say something like `testCountShouldReturnTheTotalNumberOfRecords`.  Typically there will be variations of the same functionality that you're testing, so clearly stating what the method is asserting will go along way for our future selves.

Please update these test method names to be more descriptive.

Reply:
I agree. 

Resolved: 7733a601